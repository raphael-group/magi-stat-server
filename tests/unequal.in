{
    "Y": [
        "female", 
        "male",
		"male",
		"female" 
    ],
    "X": [
        "mutated", 
        "not mutated", 
        "not mutated", 
        "not mutated",
		"not mutated"
    ]
}