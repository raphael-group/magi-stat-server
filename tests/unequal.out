{
    "Error": [
        "X and Y are unequal lengths"
    ]
}
